jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -366,6 +366,7 @@ def test_shutter_state(open_shutters, main_shutter, result):
     assert shutter_state == result
 
 
+@pytest.mark.xfail(reason="Corrected reference after CRDS prefetch change producing wrong slit count.")
```

Comment:
👍

Reply:
Thanks.   I will note that I cannot merge this,  so if there is no objection,  someone please merge this in now.   (Fingers crossed,  it should at least avoid future incorrect assignments like this one.)

Reply:
You should be able to merge now.

Reply:
Nadia added me to maintainers,  by the power vested in me as CRDS-dude,  I'm merging,  fingers crossed,  unless someone else does first.